packages/main/src/types/ToolbarStyling.ts

```diff
@@ -0,0 +1,26 @@
+/**
+* Defines the design of ui5-toolbar background.
```

Comment:
The description suggests it's for the background. Then I see it's about with or without a border.